server/backends/blobstore/blobstore.go

```diff
@@ -44,11 +45,11 @@ const (
 )
 
 // Returns whatever blobstore is specified in the config.
-func GetConfiguredBlobstore(c *config.Configurator) (interfaces.Blobstore, error) {
+func GetConfiguredBlobstore(ctx context.Context, c *config.Configurator) (interfaces.Blobstore, error) {
 	log.Debug("Configuring blobstore")
 	if c.GetStorageDiskRootDir() != "" {
 		log.Debug("Disk blobstore configured")
-		return NewDiskBlobStore(c.GetStorageDiskRootDir())
+		return NewDiskBlobStore(ctx, c.GetStorageDiskRootDir())
```

Comment:
Can we skip this one? I think the server won't init if it's slow / doesn't work, and it only happens once, so don't think it's worth trying to trace on the root context.

Reply:
np, done.